vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -1,226 +0,0 @@
-export REGISTRY='{{ REGISTRY }}'
-export IRONIC_HOST_IP='{{ IRONIC_HOST_IP }}'
-export DEPLOY_KERNEL_URL='{{ DEPLOY_KERNEL_URL }}'
-export DEPLOY_RAMDISK_URL='{{ DEPLOY_RAMDISK_URL }}'
-export IRONIC_URL='{{ IRONIC_URL }}'
-export IRONIC_INSPECTOR_URL='{{ IRONIC_INSPECTOR_URL }}'
-export POD_CIDR='{{ POD_CIDR }}'
-export SERVICE_CIDR='{{ SERVICE_CIDR }}'
-export API_ENDPOINT_HOST='{{ CLUSTER_APIENDPOINT_HOST }}'
-export API_ENDPOINT_PORT='{{ API_ENDPOINT_PORT }}'
-export IMAGE_URL='{{ IMAGE_URL }}'
-export IMAGE_CHECKSUM='{{ IMAGE_CHECKSUM }}'
-export IMAGE_CHECKSUM_TYPE='md5'
-export IMAGE_FORMAT='raw'
-export PROVISIONING_POOL_RANGE_START='{{ PROVISIONING_POOL_RANGE_START }}'
-export PROVISIONING_POOL_RANGE_END='{{ PROVISIONING_POOL_RANGE_END }}'
-export PROVISIONING_CIDR='{{ PROVISIONING_CIDR }}'
-export BAREMETALV4_POOL_RANGE_START='{{ BAREMETALV4_POOL_RANGE_START }}'
-export BAREMETALV4_POOL_RANGE_END='{{ BAREMETALV4_POOL_RANGE_END }}'
-export EXTERNAL_SUBNET_V4_PREFIX='{{ EXTERNAL_SUBNET_V4_PREFIX }}'
-export EXTERNAL_SUBNET_V4_HOST='{{ EXTERNAL_SUBNET_V4_HOST }}'
-export BAREMETALV6_POOL_RANGE_START='{{ BAREMETALV6_POOL_RANGE_START }}'
-export BAREMETALV6_POOL_RANGE_END='{{ BAREMETALV6_POOL_RANGE_END }}'
-export EXTERNAL_SUBNET_V6_PREFIX='{{ EXTERNAL_SUBNET_V6_PREFIX }}'
-export EXTERNAL_SUBNET_V6_HOST='{{ EXTERNAL_SUBNET_V6_HOST }}'
-export CTLPLANE_KUBEADM_EXTRA_CONFIG="
```

Comment:
Oh what a relief to get this out of an environment variable and into its own file! :relieved:

Reply:
So, the vars are moved to a new file 
And, the commands are distributed to relevant files ?

Reply:
Previously the vars came from both this file and the Ansible vars/main.yml file. Now all vars that come from main.yml.

For these commands, these are now included as .yml files into the main templates. Which set of commands is included is dependent on OS and controlplane/worker.